### PR TITLE
DLS-3462 Welsh changes

### PR DESCRIPTION
**Pull request by gavinharris43**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -804,5 +804,21 @@ accessibility.statement.heading.seven = Technical information about this service
...
+hts.month.may = May
+hts.month.june = June
+hts.month.july = July
+hts.month.august  = August
+hts.month.september = September
+hts.month.october = October
+hts.month.november = November
+hts.month.december = December
+
+
```
Comment (lutnos) [inline conf/messages:822]: I'm confused, why are these here?

**Review by lutnos — CHANGES_REQUESTED** [review]
We should not have to have our own versions of month names in our English messages file

File: conf/messages (hunk 1 above)
Comment (gavinharris43) [reply conf/messages:822]: Removed

Comment (gavinharris43) [reply conf/messages]: Removed.

**Review by lutnos — APPROVED** [review]
Great, thanks

**Comment by lutnos** [comment]
Release requires deployment of help-to-save-frontend, blocked on  DLS-3252 Implement welsh accessibility statement file into HTS

**Pull request merged**